cedar-policy-core/src/est/expr.rs

```diff
@@ -1302,13 +1355,16 @@ impl TryFrom<cst::Literal> for Expr {
             cst::Literal::False => Ok(Expr::lit(JSONValue::Bool(false))),
             cst::Literal::Num(n) => {
                 Ok(Expr::lit(JSONValue::Long(n.try_into().map_err(|_| {
-                    ParseError::ToAST("numeric literal out of range".to_string())
+                    ParseError::NumericLiteralOutOfRange(n.to_string())
```

Comment:
instead of a `String`, it would probably be more natural to have the argument be a number directly

Reply:
What if the number doesn't fit in `n` bits?

Reply:
Right.  String is good.  I was thinking that the `n` here was already a number type, but it's probably a string slice, isn't it